### DISCUSSION TITLE
Warn and locate where an image is missing the alt tag

### DISCUSSION THREAD

**Discussion by kik-o** [withastro/roadmap#751] [Proposal]
# Summary

Print out where in the code the alt tag property is missing.

# Background & Motivation
It's difficult to locate which images do not have an alt tag. Let's make it easier.

# Goals

A **concise, bulleted-list** outlining the intended goals of this RFC. 

- Allow users to easily find which image is missing an alt tag (print line/col number)

# Example

- Warning ⚠️ "Image is missing an alt tag attribute" [24:07]

**Comment by Princesseuh** [comment]
Unfortunately, this is not easy to do, we don't currently have the precise position in the parent when a child component throws an error inside an imported function (and it's hard to know if you'd want the position in the child or in the parent in general)

Right now we show the parent file completely, which should in most cases at least scope it down a fair amount for static routes, but I agree that it's not ideal.

It's not impossible work either, but it's not scoped to this alt error

**Comment by florian-lefebvre** [comment]
it's possible using https://ota-meshi.github.io/eslint-plugin-astro/ or the dev toolbar audit: https://docs.astro.build/en/guides/dev-toolbar/#audit
